src/components/IndiaStateMap/Switch.js

```diff
@@ -10,15 +10,15 @@ const Switch = ({ mode, setMode, activeAttribute }) => {
           className={
             "btn btn-chart " + (mode === 'normal' ? activeAttribute : "")
           }
-          onClick={() => { setMode('normal') }}
+          onClick={() => { changeMode('normal', 'confirmed') }}
```

Comment:
I think we need to change `'confirmed'` to `activeAttribute`. Otherwise it will reset back to confirmed instead of whatever is the activeAttribute (On Clicking the buttons)

Check the deployment preview, I was able to reproduce this issue.